src/DeserializationPojo/ListUsersDataPojo.java

```diff
@@ -1,14 +1,12 @@
-package pojo2;
+package DeserializationPojo;
 
-public class data2 {
-	
-	
+public class ListUsersDataPojo {
```

Comment:
Better name `UserPojo`. Choose singular or plural and stick to it across all class names.